### PR TITLE
click anywhere on answer selection to toggle checkbox

### PR DESCRIPTION
**Pull request by werner33**

### CODE REVIEW

File: codeTykeReact/src/components/QuestionSelection.js [hunk 1]
```diff
@@ -5,20 +5,21 @@ const QuestionSelection = (props) => {
   const handleCheckbox=(event, currentSelectionIndex)=>{
     let newCheckboxStatus = props.checkboxStatus.map((status, index)=>{
       if(index === currentSelectionIndex){
-        return event.target.checked;
+        return status = !status;
```
Comment (brightcodedev) [inline codeTykeReact/src/components/QuestionSelection.js]: Is this supposed to be a conditional? It's currently assigning.

File: codeTykeReact/src/App.js [hunk 2]
```diff
@@ -2,14 +2,17 @@ import React from 'react';
...
 function App() {
+
+  var learningModule = <LearningModule />
+
   return (
     <div>
       <Navbar />
       <div id="mainWrapper">
-        <LearningModule />
+        {learningModule}
```
Comment (brightcodedev) [inline codeTykeReact/src/App.js]: Are you doing this step as a preparation for something later?

File: codeTykeReact/src/components/LearningModule.js [hunk 3]
```diff
@@ -13,15 +13,15 @@ const LearningModule = (props) => {
...
-    }).catch((err)=>{
-      console.log(err);
-    })
+      fetch("http://localhost:8080/problems/"+questionId)
+      .then((res)=>{
+        return res.json();
+      }).then((data)=>{
+        setCurrentQuestion(data);
+        setLoading(false);
+      })
```
Comment (brightcodedev) [inline codeTykeReact/src/components/LearningModule.js]: Don't forget to add the error handling back in.

File: codeTykeReact/src/components/QuestionSelection.js (hunk 1 above)
Comment (werner33) [reply codeTykeReact/src/components/QuestionSelection.js]: I don't understand what you mean here. It sets status to a boolean. Toggles it if that was the answer clicked on.

File: codeTykeReact/src/App.js (hunk 2 above)
Comment (werner33) [reply codeTykeReact/src/App.js]: Changed.

File: codeTykeReact/src/components/LearningModule.js (hunk 3 above)
Comment (werner33) [reply codeTykeReact/src/components/LearningModule.js]: Added.

**Pull request merged**